stable_baselines3/common/env_util.py

```diff
@@ -74,14 +74,18 @@ def make_vec_env(
     :return: The wrapped environment
     """
     env_kwargs = {} if env_kwargs is None else env_kwargs
+
     vec_env_kwargs = {} if vec_env_kwargs is None else vec_env_kwargs
     monitor_kwargs = {} if monitor_kwargs is None else monitor_kwargs
     wrapper_kwargs = {} if wrapper_kwargs is None else wrapper_kwargs
 
     def make_env(rank):
         def _init():
             if isinstance(env_id, str):
-                env = gym.make(env_id, **env_kwargs)
+                # if the render mode was not specified, we set it to `rgb_array` as default.
```

Comment:
I need to check, but is `render_mode` argument part of the gym api?

If not, then we should do that unless we know this argument exists.

Reply:
@araffin you're right, it is not required for each class to have this argument in the init function (it is a guaranteed to exist though, see https://github.com/Farama-Foundation/Gymnasium/blob/f037fdea5a0adca5573aa5509bdc77bf48603416/gymnasium/core.py#L51) 
But their migration says this is the way to set the render mode. 

We could wrap it with a try except block to accomodate for environments that don't allow to specify the render mode?

Reply:
> it is a guaranteed to exist though

then should be fine (we might revert it back later in case it causes too much troubles)